tiup: add 3 docs component-help/prune/upgrade (#5221)
pingcap/docs: tiup/tiup-component-cluster-prune.md

---
title: tiup cluster prune
---

# tiup cluster prune

> When [scaling in the cluster](/tiup/tiup-component-cluster-scale-in.md), for some components, the service is not immediately stopped and the data is deleted, but you need to wait for the data scheduling to complete and then manually execute the `tiup cluster prune` command to clean up.

Reviewer: ```suggestion
When [scaling in the cluster](/tiup/tiup-component-cluster-scale-in.md), for some components, TiUP does not immediately stop their services or delete their data. You need to wait for the data scheduling to complete and then manually execute the `tiup cluster prune` command to clean up.
```

> When [scaling in the cluster]<!-- (/tiup/tiup-component-cluster-scale-in.md) -->, for some components, TiUP does not immediately stop their services or delete their data. You need to wait for the data scheduling to complete and then manually execute the `tiup cluster prune` command to clean up.

Reviewer 2: What this mean? Why comment the url?

Author: To avoid ci check, and after all prs are merged, we will add the url back.
